### PR TITLE
Store the Funding Trading Bridge smart contract as part of the umber upgrade.

### PR DESCRIPTION
**Pull request by SpicyLemon**
## Description

This PR updates the `umber` upgrade to also store a smart contract provided by Figure.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by iramiller — APPROVED** [review]
surprisingly small/tidy update to make this happen

File: app/upgrade_files/umber/funding_trading_bridge_smart_contract.wasm
Comment (SpicyLemon) [inline app/upgrade_files/umber/funding_trading_bridge_smart_contract.wasm:1]: This is just a release candidate version and will need to be updated prior to merging.

**Comment by SpicyLemon** [comment]
> surprisingly small/tidy update to make this happen

Yeah. It ended up being a lot easier than I expected.

**Review by Taztingo — APPROVED** [review]

**Comment by SpicyLemon** [comment]
I forced the umber upgraded on a mainnet quicksync and it has these lines in the log:
```
4:55PM INF Storing the Funding Trading Bridge smart contract.
4:55PM INF Smart contract stored with codeID: 39 and checksum: "ffaa4f5827f722ce12af244bf8c795217c532fbc4e3b6ff80f305cbd8f7364a3".
4:55PM INF Done storing the Funding Trading Bridge smart contract.
```

That's using the non-release version of the smart contract, though, so the actual checksum will end up being different. But it looks like it works just fine.

File: app/upgrade_files/umber/funding_trading_bridge_smart_contract.wasm
Comment (SpicyLemon) [reply app/upgrade_files/umber/funding_trading_bridge_smart_contract.wasm:1]: `v1.0.3` of the smart contract was just released, so I updated it in this PR too.

https://github.com/FigureTechnologies/funding-trading-bridge-smart-contract/releases/tag/v1.0.3

**Review by iramiller — APPROVED** [review]
Nice solution all around.

**Review by nullpointer0x00 — APPROVED** [review]

**Pull request merged**